GitHub tutorial/github_tutorial.md

```diff
@@ -70,6 +70,24 @@ Wanneer je op je eigen laptop/pc werkt aan bestanden, is "origin" de alias voor
 **upstream**
 Wanneer je op je eigen laptop/pc werkt aan bestanden, is "upstream" de alias voor de GitHub (remote) repository van VNG-Realisatie. Dit gebruik je wanneer je de lokale kopie van de repository wilt actualiseren met alle door anderen doorgevoerde wijzigingen (via een `git pull upstream master`).
 
+
+**lokale files excluden van commits** Hiervoor kan niet de .gitignore worden gebruikt omdat dit bestand ook wordt ingecheckt en dit zou er dan voor zorgen dat de GitHub Actions workflows de gegenereerde bestanden niet kan committen.
+In een git repo is er een .git/info/exclude file die hetzelfde werkt als de .gitignore. Alleen wordt deze niet gecommit waardoor het alleen voor de lokale repo geldt. De volgende regels moeten worden toegevoegd om de gegenereerde bestanden te excluden voor commit.
+
+- specificatie/genereervariant/**
+- code/**
+- test/**
+- openapitools.json
```

Comment:
Is dit niet specifiek voor HaalCentraal? Ik bedoel kan dit bestand in een andere repository niet een hele andere inhoud hebben?

Reply:
Aangegeven dat dit voorbeelden zijn.